test/ngMock/angular-mocksSpec.js

```diff
@@ -1532,6 +1532,38 @@ describe('ngMock', function() {
       });
 
 
+      it('should flush given number of pending requests beginning at specified request', function() {
+        var dontCallMe = jasmine.createSpy('dontCallMe');
+
+        hb.when('GET').respond(200, '');
+        hb('GET', '/some', null, dontCallMe);
+        hb('GET', '/some', null, callback);
+        hb('GET', '/some', null, callback);
+        hb('GET', '/some', null, dontCallMe);
+
+        hb.flush(2, 1);
+        expect(dontCallMe).not.toHaveBeenCalled();
+        expect(callback).toHaveBeenCalled();
```

Comment:
Yes it is.
(I just noticed `toHaveBeenCalled` together with `toHaveBeenCalledTimes` in test above.)